Core/HLE/sceKernelMemory.cpp

```diff
@@ -2301,3 +2301,115 @@ void Register_SysMemUserForUser()
 {
 	RegisterModule("SysMemUserForUser", ARRAY_SIZE(SysMemUserForUser), SysMemUserForUser);
 }
+
+
+struct HeapInformation : public KernelObject {
```

Comment:
I think we should:

1. Put these functions and structs in `sceKernelHeap.cpp`.
2. Keep `SysMemForKernel` here and export the functions in `sceKernelHeap.h` (kinda like sceKernelMutex.h, etc.)

-[Unknown]